tests/sentry/api/endpoints/test_organization_user_teams.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+import six
+
+from django.core.urlresolvers import reverse
+
+from sentry.testutils import APITestCase
+
+
+class OrganizationUserTeamsTest(APITestCase):
+    def test_simple(self):
+        foo = self.create_user("foo@example.com")
+        org = self.create_organization(owner=self.user)
+        team1 = self.create_team(organization=org)
+        team2 = self.create_team(organization=org)
```

Comment:
Should you have a team the member isn't part of to test that the team list is applying permissions?

Reply:
hmmm I tried to accomplish that with line 16 `self.create_team(organization=org)`, is that the incorrect way to go about it?